src/kv/RocksDBStore.h

```diff
@@ -64,6 +64,14 @@ namespace rocksdb{
 
 extern rocksdb::Logger *create_rocksdb_ceph_logger();
 
+static rocksdb::Slice make_slice(const std::optional<std::string>& bound) {
+    if (bound) {
```

Comment:
at namespace-level, `static` means that each translation unit that includes "RocksDBStore.h" would get a separate copy of the function. prefer `inline` here so they share a single definition

(were this inside a class, the `static` would have a different meaning - static member functions would be shared instead of duplicated)

also nit: use two-space indentation in the function body

Reply:
thanks for the explanation, @cbodley !